packages/augur-sdk/src/state/create-api.ts

```diff
@@ -57,11 +74,9 @@ export function buildSyncStrategies(client:Augur, db:Promise<DB>, provider: Ethe
 
     blockAndLogStreamerSyncStrategy.listenForBlockRemoved(logFilterAggregator.onBlockRemoved);
 
-    // Check on each new block to see if we need to generate a checkpoint.
-    client.events.on(SubscriptionEventName.NewBlock, async (newBlock) => {
-      const block = await provider.getBlock(newBlock.lastSyncedBlockNumber);
-      await warpController.onNewBlock(block);
-    });
+
+    if(config.useWarpSync) {
+    }
```

Comment:
Missing code here?

Reply:
Good call, This was moved to the event handler above. Removed.